### PR TITLE
Rename status-check job to 'status-check'.

### PR DESCRIPTION
**Pull request by mbfm**
Hi,
we realized all jobs would have to be specified individually, if we want to use multiple for status checks.

Instead, we want to use a single job at the end of our build workflow, which depends on all jobs that we want completed.  
For this purpose, we'd like to change the name of this job.

### CODE REVIEW

**Comment by netomi** [comment]
I double checked and there is no check with name "status-check" available in this repo. Maybe you refer to the step build in build.yml that depends on test and legal?

**Comment by mbfm** [comment]
Yes, we had it named 'build' to test it with the existing OtterDog configuration. We plan to rename it with this pull request, once the OtterDog config matches: https://github.com/eclipse-opendut/opendut/pull/22

Unfortunately, GitHub seems to currently have problems, which is why the build is failing...

**Comment by netomi** [comment]
Alright, thanks for clarifying, the change does look ok then. I also noticed GitHub outages atm, according to their status page, actions are in error state.

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
I will apply the change once the referenced PR is merged.

**Comment by mbfm** [comment]
> I will apply the change once the referenced PR is merged.

We actually can't merge the referenced PR without this configuration PR being merged, as it still requires the 'build' job to be green, which doesn't exist anymore.

Theoretically, we could work around this by having both jobs ('build' and 'status-check') at the same time, but we were hoping this configuration change PR could just be merged beforehand. Everything else in referenced PR is now ready for merging.

**Comment by netomi** [comment]
I removed the required status check for build so that you can merge.

However I think that you need to specify "build / status-check" as required status check to get the expected result afterwards imho.

**Review by kKdH — APPROVED** [review]

**Comment by kKdH** [comment]
> However I think that you need to specify "build / status-check" as required status check to get the expected result afterwards imho.

hm...we are very confused about the naming and which value we have to enter, because at first we thought we could just enter a workflow's name, but it has to be the name of a job. Then it was unclear whether it has to be just the name of a job or something like _'foo/bar'_ (and with or without spaces as displayed in the GH UI?).

So we added a job `build` to the workflow `build` and expected that the check fails because the status check is just `build` and not `build/build`:
ksnip_20240109-215752

but the status check succeeded:
ksnip_20240109-214542

So just the job name `status-check` should work.

**Comment by netomi** [comment]
tbh I am also not sure, but I double checked with the UI.

The following checks are available:

image
image

I expected that the status-check job would be accessible by "build / status-check" in the same way as "build / build" as its the job "status-check" in the workflow "build". But apparently its only available as "status-check". So the PR is looking good.

**Comment by kKdH** [comment]
Maybe the otterdog's documentation should be improved to clarify this topic, because they aren't very helpful:

ksnip_20240109-221402

**Comment by kKdH** [comment]
Due to _Source_ _Workflow_ we thought it has to/can be the name of a workflow.

**Comment by netomi** [comment]
I will try to find documentation how the status check names are derived from jobs, but that is not otterdog specific, we just use how it is defined by GitHub.

**Comment by netomi** [comment]
change is live.

**Pull request merged**
